eval/SPEC.md

```
@@ -0,0 +1,183 @@
+Your Workers → OTel spans → PubSub topic "telemetry.spans" → eval::ingest
+                                                                    ↓
+                                                              eval:spans:{fn_id} (state)
+                                                                    ↓
+                                              eval::metrics / eval::score / eval::drift / eval::report
+```
+
+The worker subscribes to `telemetry.spans` via a PubSub trigger. Every span ingested is stored in state keyed by function ID. Metrics, scoring, drift detection, and reporting read from that state on demand. A cron trigger runs drift detection periodically.
+
+## State Scopes
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)